### PR TITLE
v7.5.1

### PR DESCRIPTION
**Pull request by alexfauquette**

### CODE REVIEW

File: packages/x-license/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license",
-  "version": "7.2.0",
+  "version": "7.2.1",
```
Comment (alexfauquette) [inline packages/x-license/package.json:3]: It's because I added the `@mui/x-charts-pro` to the supported packages

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,68 @@
...
+### Docs
+
+- [docs] Clean the pages in the navbar (#13192) @flaviendelangle
+- [docs] Improve Tree View selection doc (#13105) @flaviendelangle
+- [docs] Unify Tree View `apiRef` methods doc examples (#13193) @flaviendelangle
+
+### Core
+
+- [core] Fix version in root package.json (#13159) @flaviendelangle
+- [core] Move the v6 changelogs to a dedicated file (#13183) @flaviendelangle
```
Comment (LukasTy) [inline CHANGELOG.md:55]: Nitpick: they probably don't bring any value to the user. 🙈 
```suggestion
```

**Review by LukasTy — APPROVED** [review]
LGTM. 👌

**Review by JCQuintas — APPROVED** [review]

**Review by flaviendelangle — APPROVED** [review]
Nothing to add
Not a lot of user facing stuff this week :laughing:

**Pull request merged**